label_studio/core/utils/io.py

```diff
@@ -190,17 +190,26 @@ def validate_upload_url(url, block_local_urls=True):
 
         raise LabelStudioAPIException(f"Can't resolve hostname {domain}")
 
-    if not block_local_urls:
-        return
+    if block_local_urls:
+        validate_ip(ip)
+
+
+def validate_ip(ip):
```

Comment:
Type should be `validate_ip(ip: str) -> None`

Reply:
Fixed.